Cargo.toml

```diff
@@ -14,7 +14,7 @@ rust-version = "1.63.0"
 default = ["elements"]
 elements = ["dep:elements", "bitcoin"]
 test-utils = ["simplicity-sys/test-utils"]
-serde = ["dep:serde", "bitcoin/serde", "elements/serde"]
+serde = ["dep:serde", "bitcoin/serde", "elements/serde", "elements/json-contract"]
```

Comment:
In 2a2e7fb84d29f6f315d0bc1db1c6eb584c75f5ee:

Why enable this? Let's just leave it off.

Reply:
I thought `json-contract` might be useful for some serde stuff, because the feature is enabled by default, but you're right, we probably don't need it.

Reply:
Fixed